src/lando/api/tests/test_landings.py

```diff
@@ -991,8 +941,8 @@ def test_format_patch_no_landoini(
     treestatusdouble.open_tree(repo.name)
 
     revisions = [
-        create_patch_revision(1),
-        create_patch_revision(2),
+        create_patch_revision(1, patch=None),
```

Comment:
Explicitely setting the patch to `None` will make `create_patch_revision` use the revision number to determine which NORMAL_PATCH_* to use.

Reply:
Perhaps we should add this as a comment in the code for the benefit of some future version of us. :)

Reply:
It's in the docstring of the fixture https://github.com/mozilla-conduit/lando/pull/191/files/315c3d469e73b2f2e50e738f4c5d8be0537b6e79#diff-5ae86d15f2e0c12ed398cedca1ad7cb0760e212cea88edbf9687e24f99366dc1R36-R40

Is there a better location where to move/duplicate it?

Reply:
Found a spot 😁